### PR TITLE
Unit metrics

### PR DESCRIPTION
**Pull request by rcpeene**
This is modified content from the allen sdk (link can be found in notebook). Maybe @Ahad-Allen and Ben Hardcastle could also review it as a discussion of quality control. Perhaps a QC notebook would be separate, or it might be appropriate to add here.

### CODE REVIEW

File: docs/visualization/visualize_unit_metrics.ipynb [hunk 1]
```diff
@@ -0,0 +1,472 @@
+{
```
Comment (jeromelecoq) [inline docs/visualization/visualize_unit_metrics.ipynb:1]: <p>can be "found" there</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (jeromelecoq) [inline docs/visualization/visualize_unit_metrics.ipynb:1]: <p>Would streaming makes sense here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (jeromelecoq) [inline docs/visualization/visualize_unit_metrics.ipynb:1]: <p>Can you commit the plots to the cells.? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Comment by rcpeene** [comment]
Runs on binder and passes test. Merging

**Pull request merged**